cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -81,12 +81,13 @@ private static int _main() throws UnknownHostException {
         final int port0 = Integer.parseInt(portOption.get());
 
         final int       executorCount           = Integer.parseInt(     Env.option("CASSANDRA_NODE_COUNT").or("3"));
+        final int       seedCount               = Integer.parseInt(     Env.option("CASSANDRA_SEED_COUNT").or("2"));
```

Comment:
The usual recommendation is: "2 or 3 seed nodes per DC".
But different people have different policies - so it depends on the user. But I don't expect many people to change the default value of 3.